kong/include/kong/model/target.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+option go_package = "github.com/kong/koko/internal/gen/wrpc/kong/model;model";
+
+package kong.model;
+
+import "kong/model/upstream.proto";
+
+message Target {
+  double created_at = 1;
```

Comment:
the `type` should be `int32` instead of `double`? it is defined as `int32` in all other places.

Reply:
i've forwarded the fix to the original repo (these files are copied there, ideally should be copied during build time and not on the repo.  we'll get there soon)